Test.Utility/MockModule.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using Autofac;
+
+namespace Messerli.Test.Utility
+{
+    public class MockModule : Module
+    {
+        public delegate void Register(ContainerBuilder builder);
+
+        public MockModule(List<Register> mockRegistrations)
+        {
+            _mockRegistrations = mockRegistrations;
+        }
+
+        private readonly List<Register> _mockRegistrations;
```

Comment:
Shouldn't fields come before the constructor(s)?

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/4/commits/935655401a6aa3e6a2c119a85eb99b2602eb95be